borg/archiver.py

```diff
@@ -1353,6 +1398,11 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('--keep-tag-files', dest='keep_tag_files',
                                action='store_true', default=False,
                                help='keep tag files of excluded caches/directories')
+        subparser.add_argument('--pattern', action=ArgparsePatternAction,
+                               metavar="PATTERN", help='include/exclude paths matching PATTERN')
```

Comment:
If we can't find a verb/noun-pair that makes sense and starts with a free letter we can also just make the abbreviated option non-sensical -- but then it should be really far off, so we don't block of "useful" letters, ie. things like `-w` or `-z`.